src/webui/webui.pri

```diff
@@ -29,3 +28,13 @@ SOURCES += \
     $$PWD/webui.cpp
 
 RESOURCES += $$PWD/www/webui.qrc
+
+# WebUI Translation
+WEBUI_TRANSLATIONS = $$files(www/translations/webui_*.ts)
+WEBUI_TRANSLATIONS_NOEXT = $$replace(WEBUI_TRANSLATIONS, ".ts", "")
+message("Building WebUI translations...")
+for(L, WEBUI_TRANSLATIONS_NOEXT) {
+    message("Processing $${L}")
+    system("$$QMAKE_LRELEASE -silent $${L}.ts -qm $${L}.qm")
+    !exists("$${L}.qm"): error("Building translations failed, cannot continue!")
```

Comment:
"Building **WebUI** translations failed". --> To differentiate from GUI translations and be able to accurately debug.
I wonder if we should add the failing locale name in both situations.

Reply:
>"Building WebUI translations failed". --> To differentiate from GUI translations and be able to accurately debug.

Ok. Fixed.
>I wonder if we should add the failing locale name in both situations.

It's not necessary since error message appears immediatelly after line with its name: "Processing <locale_name>".